New version: CirculatorySystemModels v0.3.1

JuliaRegistrator opened the pull request:
- Registering package: CirculatorySystemModels
- Repository: https://github.com/TS-CUBED/CirculatorySystemModels.jl
- Created by: @TS-CUBED
- Version: v0.3.1
- Commit: 0deb97dddbb0583a3316491d342c05911d31b8c5
- Reviewed by: @TS-CUBED
- Reference: https://github.com/TS-CUBED/CirculatorySystemModels.jl/issues/14#issuecomment-1868483963
- Release notes:
<!-- BEGIN RELEASE NOTES -->
`````
## Breaking changes since version 0.1.X:

1. The Double-Hill chamber `DHChamber` is now in terms of `dV/dt` rather than `dp/dt` which increases computation speed by an order of magnitude. The old form can be recovered using the `inP` argument. The parameter `Ev` has been removed. Please use the ventricular volume as an initial condition instead of the pressure. This can be calculated as $V = p/E_{min}$
2. `Compliance` and `Elastance` are now in terms of dV/dt. Use the new argument `inP=true` to recover the old form. Initial conditions need to be changed accordingly.
3. `Compliance` and `Elastance` have options for external pressure as parameter `has_ep` and as a connector pin to connect other pressures to `has_variable_ep` - the latter can take an equation like `C.ep.p ~ LV.p * K`.
4. Chamber has been deleted and replaced with `VariableCompliance` which can take a function for `E(t)` and has external pressure options like the other compliances and elastances.
5. `ShiChamber` already was formulated in terms of dV/dt. It now has the argument `inP=true` to allow the `dp/dt` form. Initial conditions need to be changed accordingly.

_*NOTE: the use of `inP` is strongly discouraged, since it needs the first derivative of the Elastance function, is less stable, and an order of magnitude slower. If you want to define initial condtions in pressure, it is advised to calculate the corresponding initial volume.*_
`````
<!-- END RELEASE NOTES -->

<!-- bf0c69308befbd3ccf2cc956ac8a4671113b158fd87a6d4e741b0173dab051a48f7dc43e593ffff4357bc647a3a83598f75523f6d086c6bd01e8df6233821a965df7469803a8d66610ef1f7b66f8c210395b379b568b54f773981bde4b42ead1e0d3f48c3ae7a8c6162d626b3fcd3d607c9ca5230e0b61b55fbdcaa91bd5d2da458db7fef880601698a49e15b36739d4abe386cb1ae1552b1d5996e0744f07e6d16a5f25c7711289090babac718e8c500f0bc5c4f611155bf533faa0316018bb -->

TS-CUBED commented:
This seems to be a problem with NonLinearSolve if I read the stacktrace correctly?

Anything I can do?

JuliaRegistrator committed:
New version: CirculatorySystemModels v0.3.1

UUID: 4211d73e-f4e8-40c4-b600-92e4b82f0e1a
Repo: https://github.com/TS-CUBED/CirculatorySystemModels.jl.git
Tree: de472e62b5063f393a132bbf13ba246c2c5b2d3b

Registrator tree SHA: 17aec322677d9b81cdd6b9b9236b09a3f1374c6a
